python/build_wheel.py

```diff
@@ -108,17 +104,18 @@ def sed(pattern, replace, source, dest=None):
 
     shutil.copyfile("LICENSE.txt", os.path.join(FLAGS.whl_dir, "LICENSE.txt"))
     shutil.copyfile("setup.py", os.path.join(FLAGS.whl_dir, "setup.py"))
+    shutil.copyfile("pyproject.toml", os.path.join(FLAGS.whl_dir, "pyproject.toml"))
 
     os.chdir(FLAGS.whl_dir)
     print("=== Building wheel")
-    args = ["python3", "setup.py", "bdist_wheel"]
+    args = ["python3", "-m", "build"]
```

Comment:
Not sure how to make sure the `build` package is installed in the containers.

Reply:
TODO: I need to double check this

Reply:
Started pipeline: 21638660

Reply:
Yep will need to install this in build container too

> /usr/bin/python3: No module named build
> error: Building wheel failed failed

I believe that would go here for RHEL and Ubuntu builds:
- https://github.com/triton-inference-server/server/blob/0194c3da9ddeeff07547f46aa058cf88acb51893/build.py#L987-L992
- https://github.com/triton-inference-server/server/blob/0194c3da9ddeeff07547f46aa058cf88acb51893/build.py#L1107-L1109

Reply:
I will try this out.